book/src/user/shielded-scan.md

```diff
@@ -1,90 +1,75 @@
 # Zebra Shielded Scanning
 
-This document describes Zebra's shielded scanning from users' perspective.
+The `zebra-scanner` binary is a standalone application that utilizes Zebra libraries to scan for transactions associated with specific Sapling viewing keys. It stores the discovered transactions and scanning progress data in a RocksDB database.
 
+For this application to function, it requires a `zebrad` node to be running concurrently with the scanner application.
+ 
 For now, we only support Sapling, and only store transaction IDs in the scanner results database.
-Ongoing development is tracked in issue [#7728](https://github.com/ZcashFoundation/zebra/issues/7728).
+
+Ongoing development is tracked in issue [#7728](https://github.com/ZcashFoundation/zebra/issues/7728) and [#8573](https://github.com/ZcashFoundation/zebra/issues/8573).
 
 ## Important Security Warning
 
 Zebra's shielded scanning feature has known security issues. It is for experimental use only.
 
-Do not use regular or sensitive viewing keys with Zebra's experimental scanning
-feature. Do not use this feature on a shared machine. We suggest generating new
-keys for experimental use or publicly known keys.
+Do not use regular or sensitive viewing keys with Zebra's experimental scanning feature. Do not use this feature on a shared machine. We suggest generating new keys for experimental use or publicly known keys.
 
 ## Build & Install
 
-Use [Zebra 1.6.0](https://github.com/ZcashFoundation/zebra/releases/tag/v1.6.0)
-or greater, or the `main` branch to get the latest features, and enable the
-`shielded-scan` feature during the build. You can also use Rust's `cargo` to
-install the latest release:
+Use [Zebra 1.9.0](https://github.com/ZcashFoundation/zebra/releases/tag/v1.9.0) or greater, or the `main` branch to get the latest features.
+
+You can also use Rust's `cargo` to install `zebra-scanner` from the latest release Zebra repository:
 
 ```bash
-cargo install --features shielded-scan --locked --git https://github.com/ZcashFoundation/zebra zebrad
+cargo install --locked --git https://github.com/ZcashFoundation/zebra zebra-scan
```

Comment:
This probably also installs `zebrad` (as `zebrad-for-scanner`), we may want to hide that behind a feature later.